### PR TITLE
Adding a kafka producer to the receptor controller

### PR DESCRIPTION
**Pull request by Jason-RH**
Responses from the websocket connection (only payload messages) will be published to kafka. These messages are keyed on the account number and the message id. This is the same message id that is passed to receptor and returned by the /jobs endpoint.

Need to confirm what information we want in the kafka message (RoutingInfo/Data).

### CODE REVIEW

File: internal/controller/ws/client.go [hunk 1]
```diff
@@ -196,6 +204,36 @@ func (c *rcClient) consume(ctx context.Context) {
 	}
 }
 
+func (c *rcClient) produce(ctx context.Context, m protocol.Message, kw *kafka.Writer) error {
```
Comment (dehort) [inline internal/controller/ws/client.go]: @Jason-RH I submitted a PR (PR #39) with some suggested changes to this method.  Please take a look and let me know what you think!  Let me know if we need to discuss.

File: go.mod [hunk 2]
```diff
@@ -14,6 +14,7 @@ require (
 	github.com/segmentio/kafka-go v0.3.4
 	github.com/spf13/viper v1.6.1
 	github.com/stretchr/testify v1.4.0
+	github.com/tidwall/gjson v1.4.0
```
[the remark this answers is not in the document]
Comment (Jason-RH) [reply go.mod]: It was only used to obtain json values so no I don't think we'll need it with your changes.

File: internal/controller/ws/client.go (hunk 1 above)
Comment (Jason-RH) [reply internal/controller/ws/client.go]: merged it in.

Comment (Jason-RH) [reply internal/controller/ws/client.go]: Thanks!

**Comment by Jason-RH** [comment]
@dehort Updated after your request changes!

**Pull request merged**